operator/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.elastic.co/wolfi/python:3.12-dev AS build
```

Comment:
If possible, let's pin the version with the format

```
FROM docker.elastic.co/wolfi/python:3.12-dev@sha256:****** AS build
```

Reply:
I have mixed feelings, every time I've built it, it had a different hash, would like not to be submerged by renovate updates :sweat_smile:

Reply:
Gotcha, I need to find out what guidelines are the ones when using the chainguard docker images.

Reply:
Added the digest